lib/rpmsg/rpmsg_virtio.c

```diff
@@ -545,6 +555,10 @@ static int rpmsg_virtio_ns_callback(struct rpmsg_endpoint *ept, void *data,
 	if (len != sizeof(*ns_msg))
 		/* Returns as the message is corrupted */
 		return RPMSG_SUCCESS;
+
+#ifdef VIRTIO_CACHED_BUFFERS
```

Comment:
Seem not needed, already done in rpmsg_virtio_get_rx_buffer, function is an endpoint callback

Reply:
ACK